### DISCUSSION TITLE
Very pleased to find this app!!!

### DISCUSSION THREAD

**Discussion by FlavioTheoLima** [ezra-bible-app/ezra-bible-app#676] [General]
I was looking for a long time for a Bible app to install on Linux and I didn't find one that I liked, but God enlightened my mind and brought me Ezra Bible... I am grateful to God for using Tobias Klein to develop this fantastic app !
I'm from Brazil... I don't speak English well.

**Comment by tobias-klein** [comment]
Thank you! I am glad the app is useful for your Bible studies! :) And thanks to @debritto for the Brazilian Portuguese translation of the user interface!

**Reply by FlavioTheoLima** [reply]
God bless everyone!
